modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -279,148 +286,258 @@ public function createOrderTableIfNotExist() {
 							$table->float('value');
 							$table->integer('bits');
 							$table->integer('bits_payed');
-							$table->text('flyp_id');
+							$table->string('blockonomics_currency');
```

Comment:
Create new table `blockonomics_orders` . Let us not worry about migration. 
- We will not have `id` column
- `addr` will be primary_key
- add index to `id_order` column (https://laravel.com/docs/5.2/migrations#creating-indexes )

Reply:
Updated in ffcbf89 and 2bc64ea